core/src/mindustry/world/blocks/production/Drill.java

```diff
@@ -124,6 +124,12 @@ public void drawPlace(int x, int y, int rotation, boolean valid){
             Draw.rect(returnItem.icon(Cicon.small), dx, dy - 1);
             Draw.reset();
             Draw.rect(returnItem.icon(Cicon.small), dx, dy);
+
+            if(drawMineItem){
+                Draw.color(returnItem.color);
+                Draw.rect("drill-top", tile.worldx() + offset, tile.worldy() + offset);
```

Comment:
patch for in case #4056 gets merged (separate pull request since technically its 2 different things entirely)

```suggestion
                Draw.rect(itemRegion, tile.worldx() + offset, tile.worldy() + offset);
```